code/modules/admin/admin.dm

```diff
@@ -917,6 +912,9 @@ var/global/floorIsLava = 0
 	set desc="Globally Toggles OOC"
 	set name="Toggle OOC"
 
+	var/confirm = alert(src, "[ooc_allowed ? "Do you hate fun and want to turn off OOC?" : "Are you a wonderful person and want to re-enable OOC?"]", "Confirm", "Yes", "No")
```

Comment:
Only Satan turns off OOC.

Also you don't even log in anymore celt, not an issue :^).